copyRawDataToDiagnosticsCluster.sh

```diff
@@ -96,6 +96,13 @@ do
 	then
 		mkdir -p ${RAWDATADIR}/${filePrefix}/Info
 		echo "Copying data to DiagnosticsCluster.." >> $LOGGER
+		printf "project,group,demultiplexing,copy_data,which_pipeline,copy_prm\n" > $LOGDIR/${filePrefix}/${filePrefix}.uploading
```

Comment:
The header should contain these attributes:
run_id, group, demultiplexing, copy_raw, projects, date